pkg/controller/account/ec2.go

```diff
@@ -19,7 +19,9 @@ import (
 
 // InitializeSupportedRegions concurrently calls InitializeRegion to create instances in all supported regions
 // This should ensure we don't see any AWS API "PendingVerification" errors when launching instances
-func (r *ReconcileAccount) InitializeSupportedRegions(reqLogger logr.Logger, account *awsv1alpha1.Account, regions map[string]map[string]string, creds *sts.AssumeRoleOutput) error {
+// NOTE: This function does not have any returns. In particular, error conditions from the
+// goroutines are logged, but do not result in a failure up the stack.
+func (r *ReconcileAccount) InitializeSupportedRegions(reqLogger logr.Logger, account *awsv1alpha1.Account, regions map[string]map[string]string, creds *sts.AssumeRoleOutput) {
```

Comment:
Note to reviewers: Taking the return out of the prototype here is a logical no-op, since as written this could only ever return `nil`. I did it to be explicit, and so future me isn't confused by the redundant error check in the caller.